include/dpp/message.h

```diff
@@ -956,7 +956,9 @@ enum message_flags {
 	/// this message is only visible to the user who invoked the Interaction
 	m_ephemeral = 1 << 6,
 	/// this message is an Interaction Response and the bot is "thinking"
-	m_loading = 1 << 7
+	m_loading = 1 << 7,
+	/// this message failed to mention some roles and add their members to the thread
+	m_failed_to_mention_some_roles_in_thread = 1 << 8,
```

Comment:
maybe m_thread_mention_failed is better? m_failed_to_mention_some_roles_in_thread is a massive mouthful...

Reply:
okay. changed!